lessons/operator/03/README.md

```diff
@@ -1,370 +1,212 @@
-# Lesson 3: Introduction to Sensu Go
-
-- [Web App](#web-app)
-  - [Dashboard](#dashboard)
-  - [Namespaces switcher](#namespaces-switcher)
-  - [Event list & event detail views](#event-list--event-detail-views)
-  - [Entity list & entity detail views](#entity-list--entity-detail-views)
-  - [Silencing](#silencing)
-  - [Configuration](#configuration)
-- [EXERCISE 1: log in to the Sensu web app](#exercise-1-log-in-to-the-sensu-web-app)
-- [Command Line Interface (CLI)](#command-line-interface--cli)
-  - [API client](#api-client)
-  - [Configuration management](#configuration-management)
-  - [Inventory](#inventory)
-  - [Output formats](#output-formats)
-  - [Interactive and non-interactive modes](#interactive-and-non-interactive-modes)
-- [EXERCISE 2: install and configure `sensuctl`](#exercise-2-install-and-configure-sensuctl)
-- [EXERCISE 3: use the `sensuctl --help` command](#exercise-3-use-the-sensuctl---help-command)
-- [EXERCISE 4: inspect an event in JSON format](#exercise-4-inspect-an-event-in-json-format)
-- [EXERCISE 5: explore Sensu resources using `sensuctl`](#exercise-5-explore-sensu-resources-using-sensuctl)
-- [EXERCISE 6: create an API Key for personal use](#exercise-6-create-an-api-key-for-personal-use)
-- [EXERCISE 7: export Sensu resources using `sensuctl dump`](#exercise-7-export-sensu-resources-using-sensuctl-dump)
+# Lesson 3: Using the Sensu CLI
```

Comment:
Yeah, I mostly put them together because functionally it makes sense to flow them that way, though I do agree that the API Key generation feels a bit premature here.

Reply:
Keeping as-is for now., but worth revisiting overall flow on a second pass, post-Illuminate.